plugins/CustomDimensions/Archiver.php

```diff
@@ -195,6 +217,19 @@ public function queryCustomDimensionActions(DataArray $dataArray, $valueField, $
         // get query with segmentation
         $logAggregator = $this->getLogAggregator();
         $query     = $logAggregator->generateQuery($select, $from, $where, $groupBy, $orderBy);
+
+        if ($this->rankingQueryLimit > 0) {
+            $rankingQuery = new RankingQuery($this->rankingQueryLimit);
+            $rankingQuery->addLabelColumn(array($valueField, 'url'));
+            // TODO: combine below 4
+            $rankingQuery->addColumn(Metrics::INDEX_PAGE_SUM_TIME_SPENT, 'sum');
+            $rankingQuery->addColumn(Metrics::INDEX_BOUNCE_COUNT, 'sum');
+            $rankingQuery->addColumn(Metrics::INDEX_PAGE_EXIT_NB_VISITS, 'sum');
```

Comment:
Guess those line can be removed, as they are actually contained in `$metricIds`?

Reply:
:+1: